src/widgets/totalHrsAndGranteeGraph.js

```diff
@@ -14,25 +14,36 @@ function addOrUpdateResponse(traceIndex, res, xValue, valueToAdd) {
 
   const valueToUse = parseFloat(cleanValue, 10) === 0 ? 0 : parseFloat(cleanValue, 10);
 
-  if (res[traceIndex].x.includes(xValue)) {
-    // Update existing Y value.
-    const indexToUpdate = res[traceIndex].x.indexOf(xValue);
-    res[traceIndex].y[indexToUpdate] += valueToUse;
-  } else {
-    // Add X value and Update Y value.
-    res[traceIndex].x.push(xValue);
-    res[traceIndex].y.push(valueToUse);
-  }
+  res.forEach((responseObject, index) => {
```

Comment:
Seems this is a 180 on the conversation we had about creating an entry with all possible X values vs adding 0 entries.